components/suggestion/command_provider/app_command_parser/app_command_parser.test.ts

```diff
@@ -765,7 +760,7 @@ describe('AppCommandParser', () => {
                     Description: 'The Jira issue summary',
                     Hint: '',
                     IconData: 'Create icon',
-                    Suggestion: 'Sum',
+                    Suggestion: 'summary: `Sum`',
```

Comment:
Should we maybe use `ModalLabel` here if it exists? It's more of a "proper" label for the field

Reply:
I have mixed feelings, because it may create some discrepancies. A field that is:
```
{
    location: 'ticket-name'
    label: 'ticket-name'
    modalLabel: 'Ticket Name`
}
```
will have you write ticket-name, and then show `Ticket Name: ""`. I don't know. In this case it is pretty obvious, but I wonder if there is any use case where this may become more confusing. @abhijit-singh thoughts?

Reply:
@larkox Going by the example given here, I agree that we should use `ModalLabel`. If we come across a case in the future where it isn't as obvious let's flag it and revisit this if needed.